scss/bootstrap/overrides/_variables.scss

```diff
@@ -139,33 +119,29 @@ $icon-font-path: "../vendor/bootstrap/fonts/bootstrap/" !default;
 //
 //## Common layout spacings, margins, and paddings
 
-$crm-gap-small: 5px !default;
+$crm-gap-small:               5px !default;
```

Comment:
no need for all that whitespace

Reply:
This is added to keep file look good only.